third_party/open-dice/repos.bzl

```diff
@@ -0,0 +1,15 @@
+# Copyright lowRISC contributors (OpenTitan project).
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+load("//rules:repo.bzl", "http_archive_or_local")
+
+def open_dice_repos(local = None):
+    http_archive_or_local(
+        name = "open-dice",
+        local = local,
+        build_file = Label("//third_party/open-dice:BUILD.open-dice.bazel"),
+        strip_prefix = "open-dice-main",
+        urls = ["https://github.com/google/open-dice/archive/refs/heads/main.zip"],
+        sha256 = "a802fa53c3147a1ee9a06c9aaeee6d2b49f20ca6d32877f1ccfa4f9487df7f5f",
```

Comment:
Please pin this to an exact commit.  Referring to `refs/heads/main` means the archive will change every time there is a new commit to the upstream codebase and the `sha256` field will become invalid.

Use something like: `https://github.com/google/open-dice/archive/<git-hash>.tar.gz` as the URL reference.